lib/plugins/aws/package/compile/events/eventBridge/index.test.js

```diff
@@ -1334,3 +1336,121 @@ describe('AwsCompileEventBridgeEvents', () => {
     });
   });
 });
+
+describe('EventBridgeEvents', () => {
+  after(fixtures.cleanup);
+
+  describe('when using the default event bus arn', () => {
+    let cfResources;
+    let naming;
+
+    before(() =>
+      fixtures
+        .extend('function', {
+          functions: {
+            foo: {
+              events: [
+                {
+                  eventBridge: {
+                    eventBus: 'arn:aws:events:us-east-1:12345:event-bus/default',
+                    schedule: 'rate(10 minutes)',
+                  },
+                },
+              ],
+            },
+          },
+        })
+        .then(fixturePath =>
+          runServerless({
+            cwd: fixturePath,
+            cliArgs: ['package'],
+          })
+        )
+        .then(({ cfTemplate, awsNaming }) => {
+          ({ Resources: cfResources } = cfTemplate);
+          naming = awsNaming;
+        })
+    );
+
+    it('should create the correct execution role', () => {
+      const executionRolePolicies =
+        cfResources.IamRoleCustomResourcesLambdaExecution.Properties.Policies;
```

Comment:
We should use `naming` to resolve resource logical ids (then changes in them won't make test fail)